MIP41/mip41.md

```diff
@@ -142,29 +142,54 @@ MIP41c5 subproposals must use the template located at [MIP41c5-Subproposal-Templ
 
 ---
 
-### MIP41c6: Interim Facilitators and the Fallback Decision Process
+### MIP41c6: Interim Facilitators
 
 #### Interim Facilitators
 
-Whenever a Core Unit finds itself with no Facilitator, an Interim Facilitator is needed to fill any gaps. The process to chose the Interim Facilitator is called "The Fallback Decision Process" and it is detailed in the heading below. However an interim facilitator is selected, Interim facilitators should clearly state their intended time of service. A two month minimum is recommended, with the option to renew.
+Whenever a Core Unit finds itself with no Facilitator, an Interim Facilitator is needed to ensure the continuity or graceful wind-down of the Core Unit.
 
-If a Retiring Facilitator is willing, they may nominate an Interim Facilitator. Subject to the nominated Interim Facilitator's acceptance, they are ratified as the new Facilitator by a simple majority vote of all active facilitators.
+A Retiring Faciliator may nominate a single Interim Facilitator. Subject to the nominated Interim Facilitator's acceptance, they are immediately ratified as the Interim Facilitator.
```

Comment:
What's the rationale for restricting to a single interim facilitator? It's obviously easy to administer, but it decreases optionality.

Reply:
Because the incentive is the outgoing facilitator's compensation. If you have more than one you need to either double that, or give each interim facilitator half each